presto-raptor/src/main/java/com/facebook/presto/raptor/metadata/JdbcUtil.java

```diff
@@ -27,4 +38,27 @@ public static void enableStreamingResults(Statement statement)
             statement.unwrap(com.mysql.jdbc.Statement.class).enableStreamingResults();
         }
     }
+
+    public static JDBCType jdbcType(Type type)
```

Comment:
Not to say that the existing location is good, but I think is worse since it makes it appear more generic than it actually is.